presto-raptor/src/test/java/com/facebook/presto/raptor/storage/OrcTestingUtil.java

```diff
@@ -95,4 +101,56 @@ public static byte octet(int b)
         checkArgument((b >= 0) && (b <= 0xFF), "octet not in range: %s", b);
         return (byte) b;
     }
+
+    public static boolean arrayBlocksEqual(Type elementType, Block block1, Block block2)
```

Comment:
This test code depends on presto-main. Seems like these generic methods should be somewhere in there.